transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -292,7 +320,6 @@ protected UnbindFuture unbindInternal(ResourceAddress address, IoHandler handler
         return acceptor.unbind(transport);
     }
 
-
     private final IoHandler httpResourcesHandler = new IoHandlerAdapter<HttpAcceptSession>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 23 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AS1188)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AS1213)